ios/Sources/SettingFeature/SettingScreen.swift

```diff
@@ -17,61 +16,16 @@ public struct SettingScreen: View {
     }
 
     public var body: some View {
-        WithViewStore(store) { viewStore in
+        WithViewStore(store) { _ in
```

Comment:
maybe we could remove whole screen?

Reply:
I want to remain this screen 🙏🏼 
Because there is a possibility about implementing change color theme feature.

Reply:
Because we have done the language from setting, color theme could be the same I guess.
If we have time to implement this, I think it's okay to keep like this.